src/charactersheet/viewmodels/character/profile/view.html

```diff
@@ -0,0 +1,100 @@
+<card-edit-actions params="{ flip: $parent.flip }"></card-edit-actions>
+<div class="h4 card-title"> Profile </div>
+<hr class="ac-table-header" />
+<!-- ko ifnot: loaded(), completeOn: 'render' -->
+<div class="loader-wrapper"><div class="loader"></div></div>
+<!-- /ko -->
+<!-- ko if: loaded(), completeOn: 'render' -->
+<div class="panel-body row profile-view"
+    data-bind="with: entity">
+  <div class="col-sm-6 col-lg-3">
+    <h4>
+      <span class="row-title">Level:</span>
+      <strong style="font-size: 20px"class="row-data" data-bind="text: level" />
+    </h4>
+  </div>
+  <hr class="visible-xs-block" />
+  <div class="col-sm-6 col-lg-3">
+    <h4>
+      <span class="row-title">Race:</span>
+      <span class="row-data" data-bind="text: race() ? $component.shortText(race) : '-'" />
+    </h4>
+  </div>
+  <hr class="visible-xs-block visible-sm-block visible-md-block" />
+  <div class="col-sm-6 col-lg-3">
+    <h4>
+      <span class="row-title">Class:</span>
+      <span class="row-data" data-bind="text: characterClass() ? $component.shortText(characterClass) : '-'" />
+    </h4>
+  </div>
+  <hr class="visible-xs-block" />
+  <div class="col-sm-6 col-lg-3">
+    <h4>
+      <span class="row-title">Alignment:</span>
+      <span class="row-data" data-bind="text: alignment() ? $component.shortText(alignment) : '-'" />
+    </h4>
+  </div>
+  <hr class="visible-xs-block visible-sm-block visible-md-block visible-lg-block" />
+  <div class="col-sm-6 col-lg-3">
+    <div class="profile-data-row">
+      <span class="row-title">Age:</span>
+      <span class="row-data" data-bind="text: age() ? age : 'No Age'" />
```

Comment:
I think they all used to say `No {field}` when there was no value present. Was there a reason for this change?

Reply:
It was primarily a spacing thing. 'dash' communicated the same thing with less space, especially for numeric fields.